tests/Application/package.json.~1.12.0.dist

```diff
@@ -0,0 +1,58 @@
+{
```

Comment:
Do we need the package.json split between 1.12 and 1.13?

Reply:
We don't need it. I deleted it